src/common/com/intellij/plugins/haxe/model/type/HaxeTypeCompatible.java

```diff
@@ -53,7 +66,28 @@ static private boolean isAssignableFunction(SpecificFunctionReference a, Specifi
   }
 
   static private boolean isAssignableType(@NotNull SpecificHaxeClassReference a, @NotNull SpecificHaxeClassReference b) {
-    if (a.isDynamic() || b.isDynamic()) return true;
+    if (a.isDynamic() || b.isDynamic()) {
+      return true;
```

Comment:
So, is a dynamic always assignable to a more specific type?  That doesn't seem right to me.